website/content/en/docs/upgrading/compatibility.md

```diff
@@ -86,6 +86,16 @@ By adopting this practice we allow our users who are early adopters to test out
 
 ### Snapshot Releases
 
-We release a snapshot release for every commit that gets merged into the main repository. This enables our users to immediately try a new feature or fix right after it's merged rather than waiting days or weeks for release.
-Snapshot releases are suitable for testing, and troubleshooting but users should exercise great care if they decide to use them in production environments.
-Snapshot releases are tagged with the git commit hash prefixed by the Karpenter major version. For example `v0-fc17bfc89ebb30a3b102a86012b3e3992ec08adf`. For more detailed examples on how to use snapshot releases look under "Usage" in [Karpenter Helm Chart](https://gallery.ecr.aws/karpenter/karpenter).
+We release a snapshot release for every commit that gets merged into the [`aws/karpenter-provider-aws`](https://www.github.com/aws/karpenter-provider-aws). This enables our users to immediately try a new feature or fix right after it's merged rather than waiting days or weeks for release.
+
+Snapshot releases are not made available in the same public ECR repository as other release types, they are instead published to a separate private ECR repository.
+Helm charts are published to `oci://021119463062.dkr.ecr.us-east-1.amazonaws.com/karpenter/snapshot/karpenter` and are tagged with the git commit hash prefixed by the Karpenter major version (e.g. `v0-fc17bfc89ebb30a3b102a86012b3e3992ec08adf`).
```

Comment:
Should we make these values parameters in `hugo.yaml`? That would make it nice so that if we ever changed this account or region, we could just update it in one shot across all of the documentation.

Reply:
Didn't know about the hugo file, should be able to do that. Maybe this should include an update to the release workflow to update it with the current snapshot account and region automatically.